atlasdb-cassandra/src/main/java/com/palantir/atlasdb/cassandra/backup/TokenRangeFetcher.java

```diff
@@ -59,8 +59,14 @@ public TokenRangeFetcher(
 
     public Map<InetSocketAddress, RangeSet<LightweightOppToken>> getTokenRange(String tableName) {
         KeyspaceMetadata keyspaceMetadata = cqlMetadata.getKeyspaceMetadata(namespace);
-        TableMetadata tableMetadata = keyspaceMetadata.getTable(tableName);
+        if (keyspaceMetadata == null) {
+            log.error(
+                    "Could not find metadata for a keyspace that is supposed to exist",
+                    SafeArg.of("keyspace", namespace));
+            return null;
```

Comment:
Do the callers handle null here (seems like `com.palantir.atlasdb.cassandra.backup.CassandraRepairHelper#getRangesToRepair` would now NPE constructing `RangesForRepair.of(null)`? 

Should this return an empty map or explicitly throw? I'm assuming we'd want the former based on how missing table metadata is handled below 

https://github.com/palantir/atlasdb/blob/f8febe4b86401b623a0b1d45769e0b686813bc09/atlasdb-cassandra/src/main/java/com/palantir/atlasdb/cassandra/backup/TokenRangeFetcher.java#L70-L75

```suggestion
            return ImmutableMap.of();
```